components/buttons/buttons.css

```diff
@@ -0,0 +1,86 @@
+@import url(../../style-guide/compiled-style-guide.css);
+body {
+    background-color: var(--bg-color);
+    
+    font-family: 'Exo', sans-serif;
+}
+
+.component-container {
+    display: flex;
+    margin: 0.7rem;
```

Comment:
we can use px here as its <1rem

Reply:
But that value is getting applied to that element on inspecting it so found it easier to use it.